rskj-core/src/test/java/co/rsk/peg/BridgeUtilsTest.java

```diff
@@ -2093,16 +2093,16 @@ public void extractHash160FromBytes() throws BridgeIllegalArgumentException {
     @Test
     public void calculatePegoutTxSize() {
```

Comment:
we probably need more cases here as well:
- Federation of different sizes (and ERP/regular)
- what happens if no inputs or outputs are sent?
- exercising some limits (100 inputs and outputs)

Reply:
Done